docs/changelog.rst

```diff
@@ -15,7 +15,9 @@ Changelog
 * :bug:`-` Fix an edge-case problem with synchronizing database with the server.
 * :bug:`-` Users will be able to edit EVM tokens again and the information about underlying tokens will be correctly displayed.
 * :bug:`-` Fix an issue with nginx websocket proxying on docker.
-* :bug:`-` Now `Card Cashback Reversal` entries from cryptocom csv will be imported correctly.
+* :bug:`-` Now `Card Cashback Reversal` entries from cryptocom csv will be correctly imported.
+* :bug:`-` Make ethereum queries faster by removing dead open nodes. If the user has not customized the nodes they will be automatically replaced with the new ones. Otherwise a manual action is needed.
```

Comment:
For users that won't see this list updated maybe we should add something in the release notes saying what are the new rpc endpoints?

```suggestion
* :bug:`-` Users will see the list of ethereum RPC nodes updated if they didn't modify it. For users that already modified this list action is required.
```

Reply:
I am not sure why is your suggestion better? Imo it says the same and excludes "make ethereum queries faster"

Reply:
I was thinking that speed improvements ( I haven't measured) are only a side effect. The real change is updating the list of nodes.

The proposed changes focuses the attention on what was made and identifies the users affected in the same sentence. Then it inform the rest of users not affected that they will have to take action but `a manual action` is wrong afaik.

But let @LefterisJP hope and agree is fine as it is/propose a better wording here

Reply:
In general agree with Yabir. Changelog should be informative for what happened and what should the user do if anything. You are not making all ethereum queries faster withthis change, so dont' make such claims in the changelog.

Change this one to:
> Default list of open nodes has been modified. Users who used only defaults may notice an increase in speed. For others it depends on their selection and they are encouraged to review and modify their ethereum nodes list.